chef/data_bags/crowbar/migrate/nova_dashboard/031_add_multidomain.rb

```diff
@@ -0,0 +1,10 @@
+def upgrade ta, td, a, d
+  a['multidomain'] = ta['multidomain']
+
```

Comment:
nitpick: no need for the empty line, especially if you don't have it in `downgrade`.